packages/backend/src/graphql/schema.graphql

```diff
@@ -922,6 +929,13 @@ input CreateOutgoingPaymentInput {
   idempotencyKey: String
 }
 
+input CancelOutgoingPaymentInput {
+  "Outgoing payment id"
+  id: ID!
+  "Reason why this Outgoing Payment has been cancelled"
```

Comment:
We should mention something along the lines that if this outgoing payment is requested via Open Payments, the reason will be publicly visible in the `metadata` field

Reply:
Yes, I agree. Thank you.